### ISSUE TITLE
Permissions on iOS 14 / GeoJson export

### ISSUE THREAD

**Issue by sagararyal** [Plant-for-the-Planet-org/treemapper#64]
iOS 14 has strengthened permissions notifications for apps.

- [ ]  This means we need permissions to access disk to store the geojson export. [export does not work currently]

- [ ] Remove access to find and connect to devices. We do not make use of this permission currently.
  
![3478D8C2-03BD-4219-8F10-EBF482DF3877](https://user-images.githubusercontent.com/5401575/90306720-2a273d80-ded0-11ea-8a1c-75b3519a2ad1.PNG)

**Comment by norbertschuler** [comment]
> * [ ]   This means we need permissions to access disk to store the geojson export. [export does not work currently]

The "export" for me on iOS is currently just storing the file in the private local folder (this does not need any permission) of the app which is only accessible e.g. with a connected MacBook. This is the private filesystem of the app and for normal users probably less useful as when sharing the file into the iCloud drive or any other target an iOS app can share the file into.
For Android 10 the exported file cannot be stored directly into the external Download folder of the device any more ("Permission error"), so we can fall back to the private document folder like in iOS, but this again is not very useful for the user as it cannot be accessed by other app. A sharing function should be preferred here as well.
TL;DR: Replace the storing functionality with a sharing functionality?

> * [ ]  Remove access to find and connect to devices. We do not make use of this permission currently.

This does not happen to me on the emulator I tested with yet. Which device have you used?

**Comment by norbertschuler** [comment]
@sagararyal what is the purpose of the Geo-JSON-Export? If you want to share the file with other apps. e.g. Google Drive, I would propose to use https://react-native-share.github.io/react-native-share/ instead of the current solution of trying to save the file manually into the filesystem of the device with https://github.com/joltup/rn-fetch-blob.

**Comment by norbertschuler** [comment]
> > * [ ]   Remove access to find and connect to devices. We do not make use of this permission currently.
> 
> This does not happen to me on the emulator I tested with yet. Which device have you used?

@sagararyal As I can not reproduce this, I would like to close this issue.
